src/pipeline/parse.js

```diff
@@ -144,13 +138,12 @@ exports.getHighlightJS = (ast, paths) => {
     {}
   );
 
-  let js = `const rsh = require('${slash(path.join(rshPath, 'light'))}');`
-
+  let js = `var rsh = require('react-syntax-highlighter/dist/light');`
   Object.keys(languages).forEach((language) => {
     if (languageMap[language]) {
       language = languageMap[language];
     }
-    js += `\nrsh.registerLanguage('${language}', require('${slash(path.join(rshPath, 'languages', language))}').default);`
+    js += `\nrsh.registerLanguage('${language}', require('react-syntax-highlighter/dist/languages/${language}').default);`
   });
```

Comment:
The current implementation is broken if idyll is linked and a `defaultComponent` folder is used. @bclinkinbeard can you remind me in which scenario it wasn't working to have this just `require('react-syntax-highlighter')` instead of the explicit path?

One thing I did notice in my dive into browserify docs is there is a `paths` option that we can use to expand which paths browserify will use to look for modules, which may be useful for cases like this.